fecgov/fec-cms
# Feature/4610 fix read more links

## Summary

- Resolves #4610

- Remove any content that comes after the read more link

### Required reviewers
one front end reviewer

## Impacted areas of the application

modified:   fec/static/js/init.js


## How to test

- checkout `feature/4610-fix-read-more-links`
- `npm run build-js`
-  Confirm that there is no content after the Read more links on http://127.0.0.1:8000/updates/ 
(To see broken behavior, see screenshots in issue or view on production)
